### ISSUE TITLE
False positive with `no-multi-asterisks`

### ISSUE THREAD

**Issue by Drarig29** [gajus/eslint-plugin-jsdoc#773]
## Expected behavior
With the default rules (`no-multi-asterisks` enabled), the following JSDoc should pass the linting:

```js
/**
 * Some description here.
 *
 * **Example:** An example here.
 */
```

## Actual behavior

The plugin auto-fixes the JSDoc like the following:

```js
/**
 * Some description here.
 *
 * Example:** An example here.
 */
```

## ESLint Config

Just the defaults.

```js
module.exports = {
  parser: '@typescript-eslint/parser', // Specifies the ESLint parser
  parserOptions: {
    ecmaVersion: 2020, // Allows for the parsing of modern ECMAScript features
    sourceType: 'module', // Allows for the use of imports
  },
  ignorePatterns: ['dist/', 'webpack.config.js'],
  extends: [
    'plugin:@typescript-eslint/recommended',
    'plugin:jsdoc/recommended',
  ],
}
```

## ESLint sample

```js
/**
 * Some description here.
 *
 * **Example:** An example here.
 */
function foo() {

}
```

## Environment

- Node version: v14.15.5
- ESLint version: v7.31.0
- `eslint-plugin-jsdoc` version: 35.1.3

**Comment by brettz9** [comment]
We can look to overcome this for cases where the asterisk count is even, e.g.,: 4 (extra asterisks) % 2 = 0.

```js
/**
 * **bold** 
 */
```

However, I'm not sure we want to get into the harder and more error prone area of trying to fix cases like this:

```js
/**
 * Here's a list:
 * * item 1
 * * item 2 
 */
```

I think in the latter cases, we should just recommend people use the equally valid Markdown operator `-` in place of `*`. (Getting in the habit of using this operator also has the advantage that naive syntax highlighters do not consider the asterisk as the beginning of a bold statement.)

**Comment by alvis** [comment]
Can we simply only count leading/ending asterisks without spaces?
e.g. 
```ts
// OK
/**
 * Here's a list:
 * * item 1
 * * item 2 
 *  **BOLD**
 */

// FAIL
/**
 ** Here's a list:
 ** * item 1
 ** * item 2 
 ** **BOLD**
 */
```

**Comment by Drarig29** [comment]
I will work on this issue.
